CMakePresets.json

```
@@ -0,0 +1,102 @@
+{
+    "version": 3,
+    "cmakeMinimumRequired": {
+        "major": 3,
+        "minor": 22,
+        "patch": 0
+    },
+    "configurePresets": [
+        {
+            "name": "develop",
+            "description": "enable tests and examples",
+            "hidden": true,
+            "binaryDir": "${sourceDir}/build/${presetName}",
+            "cacheVariables": {
+                "alpaka_BUILD_EXAMPLES": {
+                    "type": "BOOL",
+                    "value": "ON"
+                },
+                "BUILD_TESTING": {
+                    "type": "BOOL",
+                    "value": "ON"
+                }
+            }
```

Does it need to be this verbose? In LLAMA, I am fine with something like:
```suggestion
            "cacheVariables": {
                "alpaka_BUILD_EXAMPLES": "ON",
                "BUILD_TESTING": "ON"
            }
```

---

By default `BUILD_TESTING` is off. I thought it make sense to enable tests and examples if you want to build alpaka.

---

Maybe you misunderstood me. I meant why do you need to specify the `"type"` in an extra nested JSON object. Every cache variable could just be one line, which is far easier to read IMO.